bin/rucio-admin

```diff
@@ -329,8 +330,36 @@ def set_limits(args):
     Set account limit for an account and rse.
     """
     client = get_client(args)
-    client.set_account_limit(account=args.account, rse=args.rse, bytes=args.bytes)
-    print('Set account limit for account %s on RSE %s: %s' % (args.account, args.rse, sizefmt(args.bytes, True)))
+    limit = args.limit
+    result = re.findall('^([0-9]+)([A-Z]+)$',limit)
+     
+    if limit == 'inf' or limit == 'infinity':
```

Comment:
Would be good to move this part as a method to common.utils, in case we want to use this in other methods.